### PR TITLE
Prominent notice that this repo is unmaintained

### PR DESCRIPTION
**Pull request by cben**
Both users and contributor might look at this repo and not realize Ansible itself is the new place.  (Evidence: we'd got a couple contributions here - #79, #86).
Added prominent notice to README (and a CONTRIBUTING file too which should appear when trying to open issue/PR here).

I don't want to completely close this repo to PRs/issues for now, in case someone still needs a module from here and wants to fix a bug.
I do want to signal we'll spend minimal efforts here.

@elad661 @yaacov @snecklifter Please review

I've updated #85 with current upstreaming status & interface changes.
And I'm going through PRs & issues on this repo, hope to close most.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +45,8 @@ To view a module documentation execute:
 
 ### manageiq_provider module
 
+**Replacement**: upstream https://docs.ansible.com/ansible/devel/manageiq_provider_module.html
```
Comment (elad661) [inline README.md]: nitpick: should be consistent with the messages in all other files.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# OBSOLETE — Upstreaming into Ansible itself
+
+This repository is no longer maintained.
```
Comment (elad661) [inline .github/CONTRIBUTING.md]: nitpick: I'd put "no longer maintained" in bold, so it will be more prominent.

**Review by elad661 — APPROVED** [review]
LGTM (apart from two tiny nitpicks). 👍

**Comment by snecklifter** [comment]
LGTM.

**Comment by yaacov** [comment]
@cben do you have permissions to close this repo for new PRs ?

**Review by yaacov — APPROVED** [review]
LGTM,

I think you also need to close this for new PRs

Settings -> Danger Zone -> Archive this repo

**Comment by cben** [comment]
Addressed comments.  I'll let myself merge without further review because it's just text, and don't want to forget this for another 3 weeks :-/

I don't have permissions to archive this repo, but I also don't want to archive until we're done upstreaming.

**Comment by cben** [comment]
And by now ansible.com moved my cheese, all links 404...  Fix soon.

**Pull request merged**